snapcraft/plugins/catkin.py

```diff
@@ -252,71 +229,55 @@ def build(self):
             '{}', ';'
         ])
```

Comment:
no incentive to change this as well :wink:

Nothing that can't be done with `os.walk` and `re` or `str().replace`